kube_resource_report/templates/application.html

```diff
@@ -50,6 +50,12 @@ <h2 class="subtitle">owned by team <a href="./team-{{ application.team }}.html">
         You can potentially save <strong>{{ application.slack_cost|money }} USD</strong> every month by optimizing resource requests and reducing slack.
     </div>
 
+    {% if alpha_ema < 1 %}
+    <div class="notification is-warning">
```

Comment:
I think it's not really a warning, please remove the `is-warning` (this will make it show grey) and move the whole `<div>` to the overview page (as showing it once on the homepage is enough I think).